app/models/alchemy/page/page_natures.rb

```diff
@@ -33,6 +33,15 @@ def contains_feed?
       definition["feed"]
     end
 
+    def scoped?
```

Comment:
This method name is too vague. It should reflect the setting. So `editable_by?` could work. Or matching the collection method `has_limited_editors?`